### PR TITLE
Upgrade kotsadm deployment with admin-console upgrade command

### PR DESCRIPTION
**Pull request by divolgin**

### CODE REVIEW

File: pkg/kotsadm/kotsadm.go [hunk 1]
```diff
@@ -183,6 +183,15 @@ func ensureKotsadmDeployment(deployOptions types.DeployOptions, clientset *kuber
 		}
 	}
 
+	if err = updateKotsadmDeployment(existingDeployment, deployOptions); err != nil {
+		return errors.Wrap(err, "failed to merge deployments")
+	}
+
+	_, err = clientset.AppsV1().Deployments(deployOptions.Namespace).Update(existingDeployment)
+	if err != nil {
+		return errors.Wrap(err, "failed to update api deployment")
```
Comment (marccampbell) [inline pkg/kotsadm/kotsadm.go]: name

File: pkg/kotsadm/kotsadm_objects.go [hunk 2]
```diff
@@ -109,6 +110,66 @@ func kotsadmServiceAccount(namespace string) *corev1.ServiceAccount {
...
+	var securityContext corev1.PodSecurityContext
+	if !deployOptions.IsOpenShift {
+		securityContext = corev1.PodSecurityContext{
+			RunAsUser: util.IntPointer(1001),
+		}
+	}
+
+	desiredDeployment := kotsadmDeployment(deployOptions)
+
+	// ensure the non-optional kots labels are present (added in 1.11.0)
```
Comment (marccampbell) [inline pkg/kotsadm/kotsadm_objects.go]: L123-L140 aren't needed because this deployment is new in 1.11.0

File: pkg/kotsadm/kotsadm_objects.go [hunk 3]
```diff
@@ -109,6 +110,66 @@ func kotsadmServiceAccount(namespace string) *corev1.ServiceAccount {
 	return serviceAccount
 }
 
+func updateKotsadmDeployment(deployment *appsv1.Deployment, deployOptions types.DeployOptions) error {
+	var securityContext corev1.PodSecurityContext
```
Comment (marccampbell) [inline pkg/kotsadm/kotsadm_objects.go]: I don't think this is needed for this migration

Comment (divolgin) [reply pkg/kotsadm/kotsadm_objects.go]: @marccampbell `kotsadmDeployment` function does this too... maybe leave it?

**Pull request merged**